llvm/include/llvm/IR/DebugLoc.h

```diff
@@ -142,6 +142,31 @@ namespace llvm {
     static inline DebugLoc getDropped() { return DebugLoc(); }
 #endif // LLVM_ENABLE_DEBUGLOC_COVERAGE_TRACKING
 
+    static DebugLoc getMergedLocations(ArrayRef<DebugLoc> Locs);
```

Comment:
Do these need comments explaining a) what they do and b) how they differ from DILocation::getMergedWhatever? I think updating the latter function's comment too would be helpful, pointing out when it's appropriate to use the other in both cases?

Reply:
I've tried moving the detailed comment over to the `DebugLoc` version, and added a shortened comment noting that the `DebugLoc` version is preferred - does this look good?